config/rbac/role.yaml

```diff
@@ -315,5 +315,4 @@ rules:
   - patch
   - update
   - use
-  - watch
-
+  - watch
```

Comment:
nit: Where does this change come from? A newline at the end of the last line would be good

Reply:
fixed